src/openrct2/paint/VirtualFloor.cpp

```diff
@@ -122,6 +122,11 @@ void VirtualFloorInvalidate()
         }
     }
 
+    bool existsNewRegion
```

Comment:
I think a variable name like `invalidateNewRegion` is perhaps better?

Reply:
Indeed, changed it.